ldes-fragmentisers/ldes-fragmentisers-geospatial/README.MD

```diff
@@ -9,11 +9,9 @@ This allows you to fragment the data on geolocations.
   @prefix example: <http://example.org/> .
   
   example:fragmentationStrategy [
-        a example:Fragmentation ;
-        example:name "geospatial";
-        example:maxZoomLevel { Mandatory: Required zoom level } ;
-        example:fragmenterProperty { Mandatory: defines which property will be used for bucketizing } ;
-        example:fragmenterSubjectFilter { Optional: regex to filter the subjects matching the fragmenterProperty } ;
```

Comment:
you this logic then be remove from the code?

Reply:
reverting the removal.